CASMREL-847/README.md

```diff
@@ -0,0 +1,10 @@
+# Disable JndiLookup.class in strimzi operator
+
+This hotfix applies to CSM 0.9.X (Shasta 1.4) and CSM 1.0.X (Shasta 1.5)
```

Comment:
q: take out 1.4?

Reply:
I think this applies the same to both releases because we had the same version of this operator in both releases (it has not changed in a while).   We probably just want to test it on top of a 09x system as well as on a 1.0 system.  I think the immediate need with NERSC is for csm 1.0.x.   But when we release this, it may go out in a field notice which would go out to everyone.. and if that were to be the case, we'd want to test this on top of 09x before letting it go out (assuming we leave that statement in)..